docs/plugins/rotate_asg_instances.md

```diff
@@ -9,17 +9,22 @@ After all outdated instances are shutdown successfully, it terminates them and r
 It allows gracefully shutting down each instance instead of terminating them and killing all the running processes.
 
 ## Configuration
-The plugin uses the ssh username specified in the `MOONSHOT_SSH_USER` or the `LOGNAME` environment variable for logging into the ASG instances to shutdown. The value should be the username with which you have the access to the instances. For example:
-```ruby
-export MOONSHOT_SSH_USER=abhishek.rana
-```
+The plugin uses the ssh configuration for logging into the ASG instances to shutdown. The value should be the username with which you have the access to the instances.
 
-The plugin needs no additional configuration parameters:
+The plugin accepts additional configuration parameters for ssh which should be an array:
 
 ## Example
 ```ruby
 Moonshot.config do |c|
   # ...
+  # Environment setup for ASG rotation plugin
+  c.ssh_config.ssh_user = 'abhishek.rana'
```

Comment:
I don't think this is a particularly valid example as I'm not sure that anyone is going to hard-code an SSH username.  Is an environment variable a valid option?  Can we show that?

Reply:
I don't think we need to specify any of these examples, as ssh config here is not part of the plugin. Its upto the moonshot user on how they configure their ssh using ssh_config.

Reply:
Removed the example.